CHANGELOG.md

```diff
@@ -72,7 +80,7 @@ IMPROVEMENTS:
 
 BACKWARDS INCOMPATIBILITIES / NOTES:
 
-* Resource `vcd_network` deprecated in favor of a new name `vcd_network_routed`
+* Deprecates resource `vcd_network` in favor of `vcd_network_routed`
```

Comment:
In general, I would avoid touching changelog of older versions unless really needed.

Reply:
Agree...